### ISSUE TITLE
The library doesn't seem to work

### ISSUE THREAD

**Issue by prithwirajbose** [JamesKyburz/youtube-audio-stream#125]
The library is not working. I am getting an error and finally the audio stream is not coming up. I've FFMPEG installed. Following is the error stack trace:

```
(node:15368) UnhandledPromiseRejectionWarning: RangeError: Maximum call stack size exceeded
    at Array.join (<anonymous>)
    at Object.get (d:\workspaces\audioplayer\node_modules\fluent-ffmpeg\lib\utils.js:433:24)
    at PassThrough.<anonymous> (d:\workspaces\audioplayer\node_modules\fluent-ffmpeg\lib\processor.js:500:48)
    at PassThrough.emit (events.js:327:22)
    at PassThrough.<anonymous> (d:\workspaces\audioplayer\node_modules\youtube-audio-stream\index.js:37:14)
    at PassThrough.emit (events.js:327:22)
    at PassThrough.<anonymous> (d:\workspaces\audioplayer\node_modules\youtube-audio-stream\index.js:37:14)
    at PassThrough.emit (events.js:327:22)
    at PassThrough.<anonymous> (d:\workspaces\audioplayer\node_modules\youtube-audio-stream\index.js:37:14)
    at PassThrough.emit (events.js:327:22)
 <node_internals>/internal/process/warning.js
(node:15368) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
 <node_internals>/internal/process/warning.js
```

I've tried to trace the error on line 37 of index.js. It is as following:

```
Error: Input stream error: Error parsing info: Unable to retrieve video metadata
    at PassThrough.<anonymous> (d:\workspaces\audioplayer\node_modules\fluent-ffmpeg\lib\processor.js:452:34)
    at PassThrough.emit (<node_internals>/events.js:315:20)
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {inputStreamError: Error: Error parsing info: Unable to retr…
    at getWatchPage (d:\workspaces\aud…, stack: 'Error: Input stream error: Error parsing info…ctions (internal/process/task_queues.js:93:5)', message: 'Input stream error: Error parsing info: Unable to retrieve video metadata'}
```

**Comment by baptisteArno** [comment]
Same here

**Comment by JamesKyburz** [comment]
Fixed in 0.3.7
